docs/inventory-plugins.rst

```diff
@@ -8,218 +8,120 @@ Inventory Plugins
-
-.. index::
-   pair: inventories; Red Hat Insights
-
-1. To configure a Red Hat Insights-sourced inventory, select **Red Hat Insights** from the Source field.
-
-2. The Create Source window expands with the required **Credential** field. Choose from an existing Insights Credential.
-
-3. You can optionally specify the verbosity, host filter, enabled variable/value, and update options as needed.
-
-4. Use the **Source Variables** field to override variables used by the ``insights`` inventory plugin. Enter variables using either JSON or YAML syntax. Use the radio button to toggle between the two. For a detailed description of these variables, view the `insights inventory plugin <https://cloud.redhat.com/ansible/automation-hub/repo/published/redhat/insights/content/inventory/insights>`__.
-
-
-|Inventories - create source - RH Insights example|
-
-.. |Inventories - create source - RH Insights example| image:: _static/images/inventories-create-source-insights-example.png
-   :alt: Inventories create source Red Hat Insights example
-
-.. _ug_source_openstack:
-
-OpenStack
-~~~~~~~~~~~~
-
-.. index::
-   pair: inventories; OpenStack
+An inventory that is sourced from a project means that is uses the SCM type from the project it is tied to. For example, if the project's source is from GitHub, then the inventory will use the same source.
 
+1. To configure a project-sourced inventory, select **Sourced from a Project** from the Source field.
 
-1. To configure an OpenStack-sourced inventory, select **OpenStack** from the Source field.
-
-2. The Create Source window expands with the required **Credential** field. Choose from an existing OpenStack Credential.
-
-3. You can optionally specify the verbosity, host filter, enabled variable/value, and update options as needed.
-
-4. Use the **Source Variables** field to override variables used by the ``openstack`` inventory plugin. Enter variables using either JSON or YAML syntax. Use the radio button to toggle between the two. For a detailed description of these variables, view the `openstack inventory plugin <https://docs.ansible.com/ansible/latest/collections/openstack/cloud/openstack_inventory.html>`_ in the Ansible collections documentation.
-
-|Inventories - create source - OpenStack example|
-
-.. |Inventories - create source - OpenStack example| image:: _static/images/inventories-create-source-openstack-example.png
-   :alt: Inventories create source OpenStack example
+2. The Create Source window expands with additional fields. Enter the following details:
 
-.. _ug_source_rhv:
+  - **Source Control Branch/Tag/Commit**: Optionally enter the SCM branch, tags, commit hashes, arbitrary refs, or revision number (if applicable) from the source control (Git or Subversion) to checkout. Some commit hashes and refs may not be available unless you also provide a custom ref spec in the next field. If left blank, the default is HEAD which is the last checked out Branch/Tag/Commit for this project.
```

Comment:
It's a legit term in Git: https://git-scm.com/book/en/v2/Git-Internals-The-Refspec

```suggestion
  - **Source Control Branch/Tag/Commit**: Optionally enter the SCM branch, tags, commit hashes, arbitrary refs, or revision number (if applicable) from the source control (Git or Subversion) to checkout. Some commit hashes and refs may not be available unless you also provide a custom refspec in the next field. If left blank, the default is HEAD which is the last checked out Branch/Tag/Commit for this project.
```

Reply:
I am not going to commit this suggestion because of a misspelling of branch, and changes to this file will go into separate PR. thanks for the callout on refspec.

Reply:
I fixed that misspelling in the suggested change. Also, if you are able to edit my comments, you'd be able to do that too, and then accept.